src/emcpy/plots/create_plots.py

```diff
@@ -1,13 +1,16 @@
 # This work developed by NOAA/NWS/EMC under the Apache 2.0 license.
 import os
+import emcpy
 import numpy as np
 import matplotlib
 import matplotlib.pyplot as plt
 import matplotlib.gridspec as gridspec
 import cartopy.crs as ccrs
 import cartopy.feature as cfeature
+from PIL import Image
```

Comment:
Is this a new dependency or something an existing dependency already depends on?

Reply:
good question, I know it is 'wcoss approved' but not sure if we already have it in our env or not... (seems we do if the tests pass)

Reply:
Might possibly need to be added to the setup.py of eva. Maybe PIL is synonymous with pillow? Pillow is already satisfied by other packages.

Reply:
yeah it's pillow